search-service/src/main/kotlin/com/egm/stellio/search/service/AttributeInstanceService.kt

```diff
@@ -103,6 +103,7 @@ class AttributeInstanceService(
     ): Either<APIException, Unit> {
         val attributeInstance = AttributeInstance(
             temporalEntityAttribute = temporalEntityAttributeUuid,
+            time = attributeMetadata.observedAt!!,
```

Comment:
any hope to remove the last two `!!`?

Reply:
the `observedAt` contained in `attributemetadata` is a `ZoneDateTime?`. I don't see how else to do it